CliClient/locales/de_DE.po

```diff
@@ -1357,27 +1361,32 @@ msgstr "ID"
 
 #: /Users/tessus/data/work/joplin/Tools/../ElectronClient/gui/EncryptionConfigScreen.min.js:168
 msgid "Upgrade"
-msgstr ""
+msgstr "Upgrade"
 
 #: /Users/tessus/data/work/joplin/Tools/../ElectronClient/gui/EncryptionConfigScreen.min.js:181
-#, fuzzy
 msgid "Re-encrypt data"
-msgstr "Datum auswählen"
+msgstr "Daten neu verschlüsseln"
 
 #: /Users/tessus/data/work/joplin/Tools/../ElectronClient/gui/EncryptionConfigScreen.min.js:183
+#, fuzzy
 msgid ""
 "The default encryption method has been changed to a more secure one and it "
 "is recommended that you apply it to your data."
 msgstr ""
+"Die Standardverschlüsselungsmethode wurde in eine sicherere geändert, und es "
+"wird empfohlen, sie auf Ihre Daten anzuwenden."
 
 #: /Users/tessus/data/work/joplin/Tools/../ElectronClient/gui/EncryptionConfigScreen.min.js:183
 msgid ""
 "You may use the tool below to re-encrypt your data, for example if you know "
 "that some of your notes are encrypted with an obsolete encryption method."
 msgstr ""
+"Sie können das folgende Werkzeug  verwenden, um Ihre Daten erneut zu "
+"verschlüsseln, falls einige Ihrer Notizen mit einer veralteten "
+"Verschlüsselungsmethode verschlüsselt sein sollten."
 
 #: /Users/tessus/data/work/joplin/Tools/../ElectronClient/gui/EncryptionConfigScreen.min.js:185
-#, javascript-format
+#, fuzzy, javascript-format
 msgid ""
 "In order to do so, your entire data set will have to be encrypted and "
```

Comment:
Why did you add `fuzzy` there? In Poedit this is the flag `Needs Works`.

This means that the translation won't be used.